recipes/deepchecks/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "deepchecks" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepchecks-{{ version }}.tar.gz
+  sha256: ee6049bfba2bb0e863d027756c2eb29ae2b10285ac7ab7eee3cc0d8e0c9e6a58
+
+build:
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - category_encoders >=2.3.0
+    - dataclasses >=0.6
+    - ipykernel >=4.10.1
+    - ipykernel >=5.3.0  # [ py >= 38 ]
+    - ipython >=5.5.0
+    - ipywidgets >=7.5.0
+    - ipywidgets >=7.6.5  # [ py >= 38 ]
+    - matplotlib-base >=3.3.3
+    - numpy >=1.19
+    - pandas >=1.1.5
+    - pip
```

Comment:
do you really need ``pip``?

Reply:
Good catch! I had that included as it was a part of the repo's own conda recipe. But, yes, it is definitely not necessary. Thank you.

@marcelotrevisani Btw, I am huge fan of https://www.marcelotrevisani.com/grayskull. Can't thank you enough for that and grayskull! :rocket:

Reply:
Remove `pip` from recipe.

Reply:
cc: @marcelotrevisani